eng/Version.Details.xml

```diff
@@ -294,9 +294,9 @@
          aren't shipping, or those extensions packages depend on aspnetcore packages that won't ship. However, given the cost
          of maintaining this coherency path is high. This being toolset means that aspire is responsible for its own coherency.
     -->
-    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-9.0.100-preview.1" Version="9.0.0-preview.4.24218.6">
+    <Dependency Name="Microsoft.NET.Sdk.Aspire.Manifest-9.0.100-preview.1" Version="9.0.0-preview.3.24210.17">
       <Uri>https://github.com/dotnet/aspire</Uri>
-      <Sha>9d6b4ff780da64c972637368d6e1f58afef9535f</Sha>
+      <Sha>c2d83148f169bdf4980393a66eff2a28e94eb6c9</Sha>
     </Dependency>
     <!-- Intermediate is necessary for source build. -->
     <Dependency Name="Microsoft.SourceBuild.Intermediate.aspire" Version="9.0.0-preview.4.24218.6">
```

Comment:
You need to update this as well.  It is what is causing the prebuilt.  My suggestion is to always use darc to do these types of operations.